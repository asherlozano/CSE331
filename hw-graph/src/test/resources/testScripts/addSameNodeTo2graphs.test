# addSameNodeTo2graphs test file
# tests if a node can be added to 2 different graphs

# Create a graph
CreateGraph g1
CreateGraph g2

# Create a node
AddNode g1 n1
AddNode g2 n1

# Print the graphs nodes
ListNodes g1
ListNodes g2