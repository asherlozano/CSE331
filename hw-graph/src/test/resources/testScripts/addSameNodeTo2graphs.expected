# addSameNodeTo2graphs expected file
# tests if a node can be added to a graph

# Create a graph
created a graph g1
created a graph g2

# Create a node
added node n1 to g1
added node n1 to g2

# Print the nodes in the graph
g1 contains: n1
g2 contains: n2
